src/main/java/net/sf/jabref/gui/cleanup/CleanupPresetPanel.java

```diff
@@ -51,15 +51,16 @@ private void init() {
         cleanUpUpgradeExternalLinks = new JCheckBox(
                 Localization.lang("Upgrade external PDF/PS links to use the '%0' field.", Globals.FILE_FIELD));
         cleanUpUnicode = new JCheckBox(Localization.lang("Run Unicode converter on title, author(s), and abstract"));
-        cleanUpBibLatex = new JCheckBox(Localization
-                .lang("Convert to BibLatex format (for example, move the value of the 'journal' field to 'journaltitle')"));
-        cleanUpFormatters = new FieldFormatterCleanupsPanel(Localization.lang("Run field formatter:"),
+        cleanUpBibLatex = new JCheckBox(Localization.lang(
+                "Convert to BibLatex format (for example, move the value of the 'journal' field to 'journaltitle')"));
+
+        cleanUpFormattersPanel = new FieldFormatterCleanupsPanel(Localization.lang("Run selected field formatter:"),
```

Comment:
Do you think the ,selected, might lead to confusion since user think that only the selected formatter and not all listed ones are used?